nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -244,38 +287,71 @@ def _permuted_ols_on_chunk(
 
         # OLS regression on randomized data
         perm_scores = np.asfortranarray(
-            _t_score_with_covars_and_normalized_design(tested_vars,
-                                                       target_vars,
-                                                       confounding_vars))
-        if two_sided_test:
-            perm_scores = np.fabs(perm_scores)
-        h0_fmax_part[i] = np.nanmax(perm_scores, axis=0)
-        # find the rank of the original scores in h0_part
+            _t_score_with_covars_and_normalized_design(
+                tested_vars, target_vars, confounding_vars
+            )
+        )
+
+        if threshold is not None:
+            # TODO: Eliminate need for transpose
```

Comment:
to be done ?

Reply:
It looks like the array is transposed back and forth in a couple of places. I think the code would be cleaner if it didn't need to be, but I don't want to tackle that in this PR. Should I leave a comment as a note for future refactoring or just remove it?

Reply:
I'd just remove it.

Reply:
Done.